x-pack/plugins/transform/public/app/common/dropdown.ts

```diff
@@ -4,7 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
```

Comment:
Nit: Let's leave the line break here after the license header.

Reply:
Updated here [`15484a2` (#149879)](https://github.com/elastic/kibana/pull/149879/commits/15484a2d81ddb770cc9ffa3def370b3d46f0a2b7)